docker-compose.yml

```diff
@@ -7,6 +7,10 @@ services:
             dockerfile: Dockerfile.assets-builder
         image: elifesciences/pattern-library_assets-builder:${IMAGE_TAG}
         command: /bin/bash
+        user: "1001"
+        volumes:
+            - ./:/srv/pattern-library
+            - source_assets:/srv/pattern-library/source/assets
```

Comment:
uh, have to "unmount" node_modules here, probably

Reply:
@thewilkybarkid I don't remember where you used the syntax to allow a folder from the original image, inside a mounted volume?

Reply:
(necessary to run this on the Mac test mule, as `npm install` on the host was failing)

Reply:
https://github.com/libero/walking-skeleton/blob/fa977652b53c55c34bb726203cc1ff90feaf5951/docker-compose.yml#L9-L11

Reply:
any doc reference for that?

Reply:
Hmm, not really. Syntax is mentioned on https://docs.docker.com/compose/compose-file/#short-syntax-3. Can't remember where I found it.